src/lib/get-pre-release.js

```diff
@@ -0,0 +1,21 @@
+module.exports = config => {
+  const {pkg: {release = {}}, npm: {tag}} = config;
```

Comment:
You could do directly:
```js
module.exports =({pkg: {release = {}}, npm: {tag}}) => {
```
And remove `const {pkg: {release = {}}, npm: {tag}} = config;`